code/camera/camera.h

```diff
@@ -139,7 +140,8 @@ class subtitle
 public:
 	subtitle(int in_x_pos, int in_y_pos, const char* in_text = NULL, const char* in_imageanim = NULL,
 			 float in_display_time = 0, float in_fade_time = 0.0f, const color *in_text_color = NULL, int in_text_fontnum = -1,
-			 bool center_x = false, bool center_y = false, int in_width = 0, int in_height = 0, bool post_shaded=false);
+			 bool center_x = false, bool center_y = false, int in_width = 0, int in_height = 0, bool post_shaded = false,
+			 float line_height_factor = 1.0f);
```

Comment:
Please rename to `in_line_height_factor` to match the name used in the .cpp file.

Reply:
k